config/bindings/clusterspaced/kustomization.yaml

```diff
@@ -0,0 +1,9 @@
+resources:
```

Comment:
Can we use better name for the Kustomize target? I am referring to `bindings`.  At first glance I thought `bindings` refers to `RoleBinding`, but I see it contains roles, role bindings, and a service. 

It is important to use apt names because it helps readers understand better the purpose of targets. If there simply isn't a good name, it either suggests we should pick better boundaries or add good comments in these `kustomization.yaml` files.